spec/yaml/swiftnav/sbp/system.yaml

```diff
@@ -451,18 +451,18 @@ definitions:
     desc: >
       The PPS time message contains the value of the sender's local time in
       microseconds at the moment a pulse is detected on the PPS input. This
-      is to be used for syncronisation of sensor data sampled with a local
+      is to be used for synchronisation of sensor data sampled with a local
       timestamp (e.g. IMU or wheeltick messages) where GNSS time is unknown
       to the sender.
 
 
       The local time used to timestamp the PPS pulse must be generated by the
       same clock which is used to timestamp the IMU/wheel sensor data and
-      should follow the same roll-over rules.  A separate MSG_PPS_TIME
-      message should be sent for each source of sensor data which uses
-      PPS-relative timestamping.  The sender ID for each of these
-      MSG_PPS_TIME messages should match the sender ID of the respective
-      sensor data.
+      should follow the same roll-over rules (i.e. it should roll over to zero 
+      after 604800 seconds). A separate MSG_PPS_TIME message should be sent 
```

Comment:
@reimerix: Space character at the end of this line (and the one above) should be removed